app/src/main/res/values-iw/strings-iw.xml

```diff
@@ -161,9 +161,11 @@
     <string name="cancel_do_nothing">Cancel - do nothing</string>
     <string name="added">Added</string>
     <string name="checking_for_update">Checking for update..</string>
-    <string name="signal_missed">Signal Missed</string>
-    <string name="space_minutes_ago">" Minutes ago"</string>
-    <string name="space_minute_ago">" Minute ago"</string>
+    <string name="signal_missed">אין קליטה</string>
+    <string name="space_minutes_ago">" דקות"</string>
+    <string name="space_minute_ago">" דקה"</string>
+    <string name="before_minutes_ago">"לפני "</string>
+    <string name="minutes_ago">{0, choice,0# לפני {0, number, integer} דקות |1# לפני דקה|1&lt; לפני {0, number, integer} דקות }</string>
```

Comment:
What is the situation for the other supported locales? Are we going to see a regression to english there unless we have the choice message format translated for them too?

Reply:
Checked this under german and it seems that we are failing down to english.

Reply:
If we merge this PR as is then it will regress all the other locales to english for the strings which are replaced by this format string. 

To make it complete we would need to probably create a small script to recreate the format string for each other locale using the existing text we have for the plurals for those locales.